goal_src/levels/finalboss/green-eco-lurker.gc

```diff
@@ -467,9 +470,15 @@
           (s3-0 s4-0)
           )
      (while (> s3-0 0)
-      (eval-path-curve-div! (-> obj path) arg0 (the float s2-0) 'interp)
-      (if (dummy-51 obj arg0)
-       (return #t)
+      (let* ((a0-2 (-> obj path))
+             (t9-1 (method-of-object a0-2 eval-path-curve-div!))
+             (a1-1 arg0)
+             (a2-0 (the float s2-0))
+             )
+       (t9-1 a0-2 a1-1 a2-0 'interp)
+       (if (dummy-51 obj arg0 (the-as vector a2-0))
```

Comment:
I don't think this method actually use `a2-0` as an argument. Maybe we need to use `replace` on this method too